added main blurb to opalite doc

bleehu opened the pull request:
also added arm/leg description to opalites
	modified:   01_04_Species_Opaleites.txt

Issue #779 should be completed with this work. Some research with the 5e DnD book was done for this.

bleehu committed:
added main blurb to opalite doc
also added arm/leg description to opalites
	modified:   01_04_Species_Opaleites.txt

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -2,7 +2,18 @@
 
-  Opaleites are an organic amphibious species that 
+    Opaleites are an organic amphibious species that evolved on a planet that 
+was mostly covered by marshes, seas and oceans. Their unique resperatory system
```
*respiratory

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -2,7 +2,18 @@
 ==============================
 
-  Opaleites are an organic amphibious species that 
+    Opaleites are an organic amphibious species that evolved on a planet that 
```
Should we actually name Trioch here?

bleehu replied:
I think we cover it well below. I don't think it fits well here.

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -14,6 +25,12 @@ underwater sounds. Keeping the second set closed does hurt an Opaleite's speed
+    Opalites have two legs and two arms. Each leg has a foot with five webbed
+toes. Each arm has four webbed fingers and an aposable thumb. The arms also have
+wrist fins that greatly improve mobility underwater. Opalites have powerful 
+tails that usually hang down to about their knees which has a horizontal fin at
```
s-va: "which have a horizontal fin"

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -14,6 +25,12 @@ underwater sounds. Keeping the second set closed does hurt an Opaleite's speed
 while swimming. They are also used to drain the lungs when making the switch 
 from water to air.
```
The sentence fragments in the above par. are jarring, not helpful. We'll want to fix that before ship.

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -2,7 +2,18 @@
+this beautiful and diverse universe. Play as an Opalite if what you eat is very
+important to you. Play as a sub-species of Opalite if you want to dazzle the 
+world with your beautiful scales or be the toughest fish in the bay with your 
+bullet-resistant scales. 
```
The flow of this par. could use some work: The "If" sentence fragments would work a lot better as a series of clauses in one sen. (a la "schoolmates, if...you, if...universe."). Also, given the context of the doc so far, "Play as an Opalite if what you eat is very important to you" comes out of left field because it's the only mention of diet in the entire doc so far, but it's presented as a critical part of the species.

bleehu replied:
I'll re-write this as best I can, but we never got a design doc for Opaleites. So I don't really know what's supposed to go here, and I don't know if anyone on the team knows.

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -2,7 +2,18 @@
+lets them breathe under water as well as on land. An Opaleite's eyes are a 
+little better at seeing underwater then they are out of it.
+
+    Play as an Oplaite if you want to stick close to your team as if they were 
```
*Opaleite

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -2,7 +2,18 @@
+    Play as an Oplaite if you want to stick close to your team as if they were 
+your schoolmates. If your clan and where you came from is very important to you. 
+If you want to go anywhere in land, sea or air and see what there is to see in 
+this beautiful and diverse universe. Play as an Opalite if what you eat is very
```
*Opaleite

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -2,7 +2,18 @@
+your schoolmates. If your clan and where you came from is very important to you. 
+If you want to go anywhere in land, sea or air and see what there is to see in 
+this beautiful and diverse universe. Play as an Opalite if what you eat is very
+important to you. Play as a sub-species of Opalite if you want to dazzle the 
```
*Opaleite

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -2,7 +2,18 @@
+world with your beautiful scales or be the toughest fish in the bay with your 
+bullet-resistant scales. 
 
 ===== Appearance/Anatomy =====
```
Can we make the title for this section of Species files read "Appearance and Anatomy"? The "/" could imply that a) the section is about only one or the other, or b) we treat appearance and anatomy as interchangeable terms (which we shouldn't: anatomy is a scientific term focused on function, appearance is an aesthetic term focused on perception and social interaction).

bleehu replied:
If we change that here, we need to change that in the rest of them in the Races-Push branch as well. I'd rather ship this as-is and then systematically update the rest in a subsequent issue so we can side-by-side each of the species.

rosebrallier reviewed (changes requested):
Overall good progress, but the "Play as" par. needs major revision before it's shipable.
Creeped in a comment on the structure of the unedited portion of "Appearance/Anatomy"
Other notes: spelling, esp of Opaleite; grammar; a request re: section titles in Species docs

bleehu committed:
fixed grammar in Opaleite brief
 Changes to be committed:
	modified:   CharacterCreation/01_04_Species_Opaleites.txt

rosebrallier commented on CharacterCreation/01_04_Species_Opaleites.txt:
```diff
@@ -7,13 +7,16 @@ was mostly covered by marshes, seas and oceans. Their unique resperatory system
+
+    Play as a sub-species of Opaleite if you ...
+*  want to dazzle the world with your beautiful scales
+*  want to be the toughest fish in the bay with your bullet-resistant scales. 
```
I approve of the edit!

bleehu replied:
thanks! :)
